Documentation/block.md

```diff
@@ -3,7 +3,13 @@ title: Block Storage
 weight: 22
 indent: true
 ---
-
+{% assign url = page.url | split: '/' %}
+{% assign currentVersion = url[3] %}
+{% if currentVersion != 'master' %}
+{% assign branchName = currentVersion | replace: 'v', '' | prepend: 'release-' %}
+{% else %}
+{% assign branchName = currentVersion %}
+{% endif %}
```

Comment:
@travisn I have added this logic here and in another file as it was currently directly referencing a file in the repository.
This allows to get the branch name for the version the doc is from.
E.g. for `v0.8` docs the `branchName` will be `release-0.8` and for `master` branch just `master`.

Reply:
Interesting, i didn't realize we could embed javascript. 
- How can we be sure the url won't be mangled in different browsers? Or if the `http://` prefix isn't entered by the user, will the browser always add it?
- You tested it locally?

Reply:
As written on Slack, this isn't Javascript or anything done in the web browser.
This is rendered by Jekyll (Fluid template engine) which generates the docs, the url comes generated by Jekyll.